update_product_stream/README.md

```diff
@@ -127,10 +151,24 @@ Acquire the latest documentation RPM. This may include updates, corrections, and
 
 1. Download and upgrade the latest documentation RPM.
 
+   Without proxy:
+
    ```bash
    rpm -Uvh --force https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.4/noarch/docs-csm-latest.noarch.rpm
    ```
 
+   With http proxy:
+
+   ```bash
+   rpm -Uvh --force --httpproxy http://example.proxy.net --httpport 80 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.4/noarch/docs-csm-latest.noarch.rpm
+   ```
+
+   With https proxy:
```

Comment:
Do we need the http proxy section here? We are giving them a specific URL for this artifact and it is an HTTPS URL. So I think the http_proxy would just be ignored in this case.